src/roi/createRoi.m

```diff
@@ -146,6 +146,33 @@
 
       mask = createRoiLabel(mask);
 
+    case 'merge'
+
+      roiImages = specification;
+            
+      % loop through the ROIs to merge
+      for iRoi = 1:length(roiImages)
+        % load one ROI per time
+        maskToMerge = load_untouch_nii(roiImages{iRoi});
+        % take the first ROI as reference to add the others to it
+        if iRoi == 1
+            mask = maskToMerge;
+            % delete the fileprefix argument to be re-assigned with the new one
+            mask.fileprefix = [];
+        else 
+            % sum up the ROI masks
+            mask.img  = mask.img + maskToMerge.img;
+        end
+
+      end
+      
+      % check that there are no >1s 
+      mask.img(mask.img > 1) = 1;
```

Comment:
I am pretty sure that in this case you can just convert mask.img to a logical array:

```
mask.img = logical(mask.img);
```

Reply:
I think that using

```
      % merge the logical masks into one
      mask = any(mask, 4);
```

will save us from having values >1

although, maybe for the future, we would like to have different values for each single single ROI merged

Reply:
If you mean having a single image that combines different masks has a different "label" for each ROI: I agree but I would have that as an entirely different function. 
And I think that MarsBar must have some functions to do that.